### PR TITLE
oc status bare build problems

### PR DESCRIPTION
**Pull request by deads2k**
Fixes formatting (double indent) and destination resolution.

Fixes https://github.com/openshift/origin/issues/6356

@kargakis @stevekuznetsov

### CODE REVIEW

File: pkg/api/graph/test/bare-bc-can-push.yaml [hunk 1]
```diff
@@ -0,0 +1,156 @@
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    creationTimestamp: 2015-12-16T01:01:33Z
+    labels:
+      app: repo-base
+    name: repo-base
+    namespace: repo
```
Comment (kargakis) [inline pkg/api/graph/test/bare-bc-can-push.yaml]: Remove any namespaces. There are times I am running these locally and having a namespace is really annoying.

**Comment by kargakis** [comment]
one comment, LGTM otherwise

**Comment by deads2k** [comment]
[merge]

File: pkg/api/graph/test/bare-bc-can-push.yaml (hunk 1 above)
Comment (deads2k) [inline pkg/api/graph/test/bare-bc-can-push.yaml]: > Remove any namespaces. There are times I am running these locally and having a namespace is really annoying.

done

**Pull request merged**